base/gfspapp/app_options.go

```diff
@@ -225,7 +226,8 @@ func DefaultGfSpPieceStoreOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) e
 	}
 	pieceStore, err := piecestoreclient.NewStoreClient(&cfg.PieceStore)
 	if err != nil {
-		return err
+		log.Warnw("failed to new piece store", "error", err)
```

Comment:
"if not use piece store, please ignore: failed to new piece store", "error", err

Reply:
fixed